pkg/es/writer/esBulkHandler.go

```diff
@@ -223,26 +248,32 @@ func HandleBulkBody(postBody []byte, ctx *fasthttp.RequestCtx, rid uint64, myid
 	response["errors"] = overallError
 	response["items"] = items[0:inCount]
 
+	respItemsPool.Put(&origItems)
```

Comment:
nit: might be good to make this a `defer` where you `Get()` it, but it looks like this function doesn't early return anywhere, so it could be fine to keep it here

Reply:
addressed in latest commit